examples/api/main_test.go

```diff
@@ -33,7 +33,7 @@ func InitializeScenario(ctx *godog.ScenarioContext) {
 	apiFeature.RegisterSteps(ctx)
 }
 
-func TestMain(t *testing.T) {
+func TestComponent(t *testing.T) {
```

Comment:
Have we considered renaming this in the readme example?

Reply:
I think it's worth doing (I had deliberately not done so because in theory it is not 'wrong' per se, but actually when you bring it up, I think I should do it in the readme). If we do use 'TestMain' it should have the correct signature (m *testing.M);